packages/e2e/test/web-extension/.env.example

```diff
@@ -19,4 +19,5 @@ STAKE_POOL_PROVIDER_PARAMS='{"baseUrl":"http://localhost:4000/stake-pool"}'
 
 # Test Environment
 NETWORK_ID=0
-MNEMONIC_WORDS="vacant violin soft weird deliver render brief always monitor general maid smart jelly core drastic erode echo there clump dizzy card filter option defense"
+MNEMONIC_WORDS_WALLET1="vacant violin soft weird deliver render brief always monitor general maid smart jelly core drastic erode echo there clump dizzy card filter option defense"
+MNEMONIC_WORDS_WALLET2="slab gorilla reflect display cage aim silver add own arrange crew start female bitter menu inner combine exit swallow bamboo midnight wealth culture picnic"
```

Comment:
You can create two wallets from the same mnemonic. Any reason we can't just iterate the [account index](https://github.com/input-output-hk/cardano-js-sdk/pull/502/commits/6e45d265aa8db63a787cc64a373aa31ce3a0965a#diff-721f78adca74441c90355e45c0319e02940ab5b27ca53389073a76b0b2967b6aR121)

Reply:
Agree, that would be better

Reply:
I need a wallet with funds. Otherwise, I can't execute the build and sign transaction step in the test

Reply:
> I need a wallet with funds

A single mnemonic can create multiple accounts, which is all you need here to simulate multiple wallets. Maybe I'm missing some broader context?

Reply:
OK, given you're just using what [already exists](https://github.com/input-output-hk/cardano-js-sdk/tree/master/packages/e2e#local-test-network), this is fine to leave. We probably should just be using a single mnemonic with 5 accounts derived, so we could look at refactoring this later